pages/hub/releases/17-sdk-v3.3.mdx

```diff
@@ -0,0 +1,9 @@
+# 04/23 SDK v3.3
+
+[Live Betting](/hub/apps/guides/advanced/live/overview) in the `Polygon Mainnet`.
+
+You can now switch the application mode to "live" for the Polygon network. [Read more](/hub/apps/sdk/providers/live)
+
+## Updates
+- `minBet` info and new disableReason `BetslipDisableReason.BetAmountLowerThanMinBet` in [BetslipProvider](/hub/apps/sdk/providers/betslip)
+- `liveBetAmount` => `minLiveBetAmount`
```

Comment:
I suggest:

- const `liveBetAmount` marked as deprecated, because live bets are not strictly limited to "1" anymore (use `minBet` + `maxBet` from `useDetailedBetslip()` for live bet value validation).

Reply:
0ac570c82854a5cdc7b4cb9e7f870888bd25c4a1